index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="ko">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Panda_login</title>
+  <link rel="icon" href="images/logo/favicon.ico">
+  <link rel="stylesheet" href="style.css">
+</head>
+<body>
+  
+  <div class="container">
+    <div class="logo"><a href="/">
+    <img src="images/logo/logo.svg" width="396px" height="132px"></a>
+    </div>
+ 
+    <div class="login_box">
+    
+      
+    <label for="emial">이메일</label>
+    <input type="email" id="email" placeholder="codeit@email.com">
+    
+    
+    <label for="password">비밀번호</label>
+    <div class="password_wrapper">
+    <input type="password" id="password" placeholder="••••••••"> 
+    <img class="eye" src="images/icons/eye-invisible.svg" width="24px" height="24px">
```

Comment:
❗️ 수정요청  
해당 요소는 클릭 시 비밀번호가 보이고 안 보이게 만드는 요소가 될 것이므로 button요소로 작성하시는 것을 추천드려요.

```suggestion
    <button type="button"><img class="eye" src="images/icons/eye-invisible.svg" width="24px" height="24px"></button>
```